### PR TITLE
nom 5 with workaround

### PR DESCRIPTION
**Pull request by AnderEnder**

### CODE REVIEW

**Comment by AnderEnder** [comment]
LGTM

File: diamond-api/src/render_target/mod.rs [hunk 1]
```diff
@@ -16,6 +16,6 @@ mod tests {
 
     #[test]
     fn test_from_bytes() {
-        let _ex = ast::Expression::try_from(&b"template(average(emea.events\\[2019\\].clicks,n=7),skip_empty=false,none=none)|aliasByNode(1)|movingAverage(\"5min\")"[..]).unwrap();
+        let _ex = ast::Expression::try_from("template(average(emea.events\\[2019\\].clicks,n=7),skip_empty=false,none=none)|aliasByNode(1)|movingAverage(\"5min\")").unwrap();
```
Comment (andy128k) [inline diamond-api/src/render_target/mod.rs]: It contradicts to definition of test ("from bytes"). it repeats `test_from_str` now. So, it needs to be removed.

File: diamond-api/src/lib.rs [hunk 2]
```diff
@@ -1,3 +1,5 @@
+#![type_length_limit = "1519486"]
```
Comment (andy128k) [inline diamond-api/src/lib.rs]: Is it needed? I tried to compile w/o this line and it works.

**Comment by andy128k** [comment]
@AnderEnder Please squash it into single commit

**Pull request merged**